packages/block-editor/src/components/block-settings-menu-controls/index.js

```diff
@@ -31,9 +36,14 @@ const BlockSettingsMenuControlsSlot = ( { fillProps, clientIds = null } ) => {
 
 	return (
 		<Slot fillProps={ { ...fillProps, selectedBlocks } }>
-			{ ( fills ) =>
-				! isEmpty( fills ) && <MenuGroup>{ fills }</MenuGroup>
-			}
+			{ ( fills ) => {
+				return (
+					<MenuGroup>
+						{ fills }
+						<ConvertToGroupButton onClose={ fillProps?.onClose } />
```

Comment:
Did you consider adding it directly to `BlockSettingsDropdown` ? I guess it might change its position a little bit but maybe that's fine?

Reply:
Yes, but previously was in the same `MenuGroup` with the other fills and that's the reason I placed it here.

Alternatively I should create a new `MenuGroup` for `ConvertToGroupButton ` and add css to remove the border. I have the feeling that we generally don't want t0o many `borders` in such places (another example is Toolbar but don't know if it applies the same) 🤔

Reply:
Sounds good to me